test/pairwise.jl

```diff
@@ -0,0 +1,142 @@
+using Test
+using DFTK
+using LinearAlgebra
+using Random
+
+@testset "Hydrogen atom" begin
+    lattice = 16 * Diagonal(ones(3))
+    positions = [[0,0,0]]
+    charges = [1]
+    atom_types = map(ElementCoulomb, charges)
+    V(x, p) = 4*p.ε * ((p.σ/x)^12 - (p.σ/x)^6)
+    params = Dict((atom_types[1], atom_types[1]) => (; ε=1, σ=16))
+    term = DFTK.PairwisePotential(V, params)
+
+    ref = -4.365199475203764
+    γ_E = DFTK.energy_pairwise(lattice, atom_types, positions, term.V, term.params, term.max_radius)
+    @test abs(γ_E - ref) < 1e-8
+end
```

Comment:
That's a lot of tests, I'd just keep one.

Reply:
Some lines are not covered; hopes it's alright.